code/script.js

```diff
@@ -32,16 +51,98 @@ const showMessage = (message, sender) => {
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
+  showMessage(`Hi there!👩‍🍳 Welcome to Bootstrap Bakery. What's your name?`, 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
-// Set up your eventlisteners here
+const nextQuestion = (message) => {
+
+  if (questionNumber === 1) {
+    userReply(message);
+    formMessageBox.value = "";
+    setTimeout(() => question2(message), chatDelay);
+  } else if (questionNumber === 2) {
+    userReply(firstToUpperCase(message));
+    formMessageBox.value = "";
+    setTimeout(() => question3(message), chatDelay);
+  } else if (questionNumber === 3) {
+    userReply(firstToUpperCase(message));
+    formMessageBox.value = "";
+    setTimeout(() => question4(message), chatDelay);
+  }
+};
+
+const question2 = (message) => {
+  questionNumber++;
+  botReply(`Nice to meet you, ${message}. What would you like to order?`);
+
+  inputWrapper.innerHTML = `
+    <button id="cakeBtn">Cake 🎂</button> 
+    <button id="cupcakeBtn">Cupcake 🧁</button> 
+    <button id="pieBtn">Pie 🥧</button> 
+  `
+
+  document
+    .getElementById("cakeBtn")
+    .addEventListener("click", () => nextQuestion("cake"))
+  document
+    .getElementById("cupcakeBtn")
+    .addEventListener("click", () => nextQuestion("cupcake"))
+  document
+    .getElementById("pieBtn")
+    .addEventListener("click", () => nextQuestion("pie"))
+};
```

Comment:
It is a very nice and clean code! 🌈 For the next step, you can think about creating one `showQuestion` function. for example `showQuestion( botMessage, inputType, options )` where you pass your question and input type as string and options as an array. 💡